verilog/tools/lint/lint_tool_test.sh

```diff
@@ -96,6 +96,15 @@ status="$?"
   exit 1
 }
 
+echo "=== Test --show_diagnostic_context"
+"$lint_tool" "$TEST_FILE" --show_diagnostic_context > /dev/null 2> "${MY_OUTPUT_FILE}.err"
+
+status="$?"
+[[ $status == 1 ]] || {
+  echo "Expected exit code 1, but got $status"
+  exit 1
+}
+
```

Comment:
Can we add some sort of shell test that determines that an output is in fact in the particular form 
```
<file>:<line>:<col>: errmsg
```
without the diagnostic context flag and with added ^ marker with ?

Reply:
An end-to-end test here would be still good; testing if we get some expected error message.

Reply:
I prepared changes for this issue last Friday, but they somehow vanished...
I haven't checked if the commit was pushed to the branch, sorry for that.
I have updated the PR with the missing tests